hudi-client/hudi-client-common/src/main/java/org/apache/hudi/callback/common/HoodieWriteCommitCallbackMessage.java

```diff
@@ -41,13 +44,19 @@
    */
   private String basePath;
 
+  /**
+   * HoodieWriteStat.
+   */
+  private List<HoodieWriteStat> hoodieWriteStat;
+
   public HoodieWriteCommitCallbackMessage() {
   }
 
-  public HoodieWriteCommitCallbackMessage(String commitTime, String tableName, String basePath) {
+  public HoodieWriteCommitCallbackMessage(String commitTime, String tableName, String basePath, List<HoodieWriteStat> hoodieWriteStat) {
```

Comment:
Let's remove all the setters and mark all fields to be final.

Reply:
+1